cluster/charts/crossplane/README.md

```diff
@@ -119,6 +119,7 @@ and their default values.
 | `securityContextRBACManager.readOnlyRootFilesystem` | Set the RBAC Manager pod root file system as read-only. | `true` |
 | `securityContextRBACManager.runAsGroup` | The group ID used by the RBAC Manager pod. | `65532` |
 | `securityContextRBACManager.runAsUser` | The user ID used by the RBAC Manager pod. | `65532` |
+| `service.annotations` | Configure annotations on the service object. Only enabled when webhooks.enabled = true | `{}` |
```

Comment:
```suggestion
| `service.customAnnotations` | Configure annotations on the service object. Only enabled when webhooks.enabled = true | `{}` |
```

I think elsewhere this is named `customAnnotations`. See following line for example.

Reply:
ah i see i have to run`make check-diff` after `make reviewable`. Will fix that.